docs/index.markdown

```diff
@@ -15,6 +15,26 @@ If you do notice an error in this current version, feel free to [create an issue
 
 ## Copyright
 
-The original manual is likely copyright © 1979 Massachusetts Institute of Technology. It was written by S. W. Galley and Greg Pfister, but all indications point to it being work-for-hire as part of MIT's contract with DARPA.
+Copyright didn't used to be automatic, as it is in today's world.
 
-As disseminated to the United States Department of Defense, it has the [distribution statement "A"](http://www.dtic.mil/dtic/submit/distribution_statements.html), meaning that it is approved for public release and distribution is unlimited. This classification is the basis under which a good-faith effort to preserve an otherwise-public document has been made.
+Various factors need to be considered and determining the copyright status of a work can be involved: What country or countries was it published in? When was it published? Was an appropriate copyright noticed included (even though they are not required anymore, proper copyright notices used to be required and if they were either missing or inadequate in some way, it could result in loss of copyright on the work involved.) Another factor to consider is if any required filings were made with the U.S. Copyright Office, and more.
+
+Due to the various factors, which have changed over time, [Cornell's Copyright Information Center](http://copyright.cornell.edu/) publishes a guide that can be useful in helping to determine the copyright status of a work: [http://copyright.cornell.edu/resources/publicdomain.cfm](http://copyright.cornell.edu/resources/publicdomain.cfm).
+
+For some background of the MDL Programming Language document: Various versions of it were published from 1972 to at least 1979. Some had a copyright notice and some did not. Greg Pfister, the original author of this document, says the 1972 version does not. Also, the 1979 version that was submitted to the U.S. Government and that is on file with the Defense Technical Information Center, and linked to above, does not. I have also checked and this document was not registered with the U.S. Copyright Office at any point.
```

Comment:
The use of "I" here is ambiguous, since this document has multiple authors (and you, the author of this paragraph, are uncredited!).

On the other hand, the earlier references to "I" have also been ambiguous ever since my fork.

I'm not sure how best to clean this up. Probably merge in the latest commit from upstream (which clarifies Mark's position on accepting PRs) and then update the issue/PR links in the first section, at least. Then maybe change all the "I" to "we"?